README.md

```diff
@@ -15,4 +15,17 @@ steps:
     uses: reload/poc-phpcs-reviewdog-action@<VERSION>
     with:
       reviewdog_token: ${{ secrets.GITHUB_TOKEN }}`
+```
+
+A sub path can be given to the action if the composer.json file is not in the root of the working directory.
+
+Example:
+
+```yaml
+steps:
+  - name: Run phpcs
+    uses: reload/poc-phpcs-reviewdog-action@<VERSION>
```

Comment:
Oh, you need to fix the name in the examples.

Reply:
Check